warrior/Tools/reporting/html_results_template.html

```diff
@@ -622,20 +659,17 @@ <h2>Results Summary</h2>
 		<tr id="tr1" name="HeadingRow">
 			<th>Type</th>
 			<th>Name</th>
-			<th>Info</th>
+		        <th>Info</th>
+			<th>Description</th>
 			<th>Timestamp</th>
 			<th>Duration</th>
 			<th>Status</th>
 			<th>Impact</th>
-			<th>On Error</th>
-			<th>
-			<span style="padding-left:5px">Results (<i class="fa fa-line-chart">)</i></span><br/>
-			<span style="padding-left:5px">Logs (<i class="fa fa fa-book">)</i></span><br/>
-			<span style="padding-left:5px">Defects (<i class="fa fa fa-bug">)</i></span><br/>
-			</th>
-			<th colspan="6" id="summary">Summary</th>
```

Comment:
Hi @priyaarun23,

Is there a reason why you removed the "Summary" header? Please add it back.

Thanks,
Sanika

Reply:
@sanikakulkarni ,

Summary header has been removed as it was not showing up after the addition of Result, Log & Defects columns. (confirmed with the user Karthick Ekanathan) 

Added up now to make it visible for the last column of summary icon separately.